mem/ctrl/CtrlMemDynamicRTL.py

```diff
@@ -38,14 +40,19 @@ def construct(s, CtrlPktType, CtrlSignalType, ctrl_mem_size,
     num_routing_outports = num_tile_outports + num_fu_inports
 
     # Interface
+    # Stores ctrl signals into the control memory/registers.
     s.send_ctrl = SendIfcRTL(CtrlSignalType)
+    # Receives the ctrl packets from the controller.
     s.recv_pkt = RecvIfcRTL(CtrlPktType)
```

Comment:
Rename `recv_pkt` to `recv_pkt_from_controller` and `send_pkt` to `send_pkt_to_controller`.